ajax/libs/frontpress/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "frontpress",
+  "filename": "frontpress.min.js",
+  "version": "1.0.7",
+  "description": "Full front-end AngularJS template for WordPress Rest API.",
+  "keywords": [
+    "frontpress",
+    "wordpress",
+    "angular",
+    "rest",
+    "spa"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/frontpressorg/frontpress.git"
+  },
+  "author": {
+    "name": "Jota Teles",
+    "url": "https://github.com/teles"
+  },
+  "npmName": "frontpress",
+  "npmFileMap": [{
+    "basePath": "release",
+    "files": [
+      "**/*"
+    ]
+  }],
```

Comment:
This part is still wrong, would you please use tool - `./tools/fixFormat.js` to fix it for you?